File: src/IsbInterop/References/Wrappers/ReferenceInfo.vb
Namespace References.Wrappers

    ''' <summary>
    ''' Обертка над IReferenceInfo.
    ''' </summary>
    Friend Class ReferenceInfo
        Inherits CompRecordInfo
        Implements IReferenceInfo

        ''' <summary>
        ''' Конструктор.
        ''' </summary>
        ''' <param name="rcwIReferenceInfo">COM-объект IReferenceInfo.</param>
        ''' <param name="scope">Область видимости.</param>
        Public Sub New(rcwIReferenceInfo As Object, scope As IScope)
            MyBase.New(rcwIReferenceInfo, scope)
        End Sub

    End Class
End Namespace

File: src/IsbInterop.CSharp/References/Wrappers/ReferenceInfo.cs
namespace IsbInterop.References.Wrappers
{
  /// <summary>
  /// Обертка над IReferenceInfo.
  /// </summary>
  internal class ReferenceInfo : CompRecordInfo, IReferenceInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIReferenceInfo">COM-объект IReferenceInfo.</param>
    public ReferenceInfo(object rcwIReferenceInfo) : base(rcwIReferenceInfo) { }
  }
}
